actor-server/actor-frontend/src/main/scala/im/actor/server/frontend/SessionClient.scala

```diff
@@ -211,7 +213,7 @@ private[frontend] final class SessionClient(sessionRegion: SessionRegion)
       } else {
         unpack(mbBits) match {
           case Success(rawBits) ⇒
-            sessionRegion.ref ! SessionEnvelope(authId, sessionId).withHandleMessageBox(HandleMessageBox(ByteString.copyFrom(rawBits.toByteBuffer)))
+            sessionRegion.ref ! SessionEnvelope(authId, sessionId).withHandleMessageBox(HandleMessageBox(ByteString.copyFrom(rawBits.toByteBuffer), Option(remoteAddr)))
```

Comment:
Because remoteAddr is optional in  HandleMessageBox and has a default None value. Some places we don't need to pass IP with HandleMessageBox.